pegasus/sites.v3/code.org/public/music/index.haml

```diff
@@ -62,16 +62,20 @@ theme: responsive_full_width
     %h2.white=hoc_s("music_lab.quotes.heading")
     = view :"music/music_lab_carousel_quotes"
 
-%section
-  .wrapper
-    .flex-container.justify-space-between.align-items-center.wrap.gap-2
-      %figure.col-47
```

Comment:
I take it this part changed because you're making space for the additional 'Available in English only' line?

Reply:
Correct! Adding that line made the text part taller than the image so adjusting the widths of the image and text wrapper makes the section look more balanced and easier to read.

Reply:
My Kelby-senses are improving by the day

Reply:
😁 🙌